### ISSUE TITLE
`phonegap serve` should log out if it is adding the browser platform

### ISSUE THREAD

**Issue by filmaj** [phonegap/phonegap-cli#716]
If you `phonegap serve` right away after creating a project, the [browser platform will be added automatically](https://github.com/phonegap/phonegap-cli/blob/master/lib/phonegap/serve.js#L110). Even with the `-d` flag, the user is given no warning that this happening, and thus it looks like CLI is stuck not doing anything.

We should, probably even in the non-verbose case, `warn` about this happening.

Thoughts @surajpindoria ?

**Comment by surajpindoria** [comment]
Yeah, I think this is a good idea. I added an event that gets fired once the browser platform is done being created. I just took a look and it seems the decouple stuff didn't add that in, just a small change to get it going again though.

**Comment by ZaLiTHkA** [comment]
Not sure if this is worth creating a new issue for, but could somebody perhaps explain why the browser platform is added automatically with `phonegap serve`..?

**Comment by filmaj** [comment]
@ZaLiTHkA I would definitely open a new issue for this (feel free to reference this issue in there too).

**Comment by ZaLiTHkA** [comment]
Alrighty then, will do. 👍
